src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~ Licensed to the Apache Software Foundation (ASF) under one or more
+~ contributor license agreements. See the NOTICE file distributed with
+~ this work for additional information regarding copyright ownership.
+~ The ASF licenses this file to You under the Apache license, Version 2.0
+~ (the "License"); you may not use this file except in compliance with
+~ the License. You may obtain a copy of the License at
+~
+~      http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the license for the specific language governing permissions and
+~ limitations under the license.
+-->
+<Configuration status="ERROR">
+    <Properties>
+        <Property name="LOG_DIR">${env:JENKINS_HOME}/logs</Property>
```

Comment:
I'm thinking we might want to have a default value fallback in case `$JENKINS_HOME` isn't defined in the environment. Perhaps you could just use `.` as the default? Syntax is `${env:JENKINS_HOME:-.}` like in bash (sorta).

Reply:
yea... good idea. I noticed when i ran things without setting the JENKINS_HOME in my own environment it gave errors. other times it just outputted everything into my workspace rather than into a dedicated folder, which was pretty messy.